sshmux.go

```diff
@@ -60,17 +61,19 @@ type AuthRequestPassword struct {
 }
 
 type AuthResponse struct {
-	Status     string `json:"status"`
-	Address    string `json:"address"`
-	PrivateKey string `json:"private_key"`
-	Cert       string `json:"cert"`
-	Id         int    `json:"vmid"`
+	Status        string `json:"status"`
+	Address       string `json:"address"`
+	PrivateKey    string `json:"private_key"`
+	Cert          string `json:"cert"`
+	Id            int    `json:"vmid"`
+	ProxyProtocol *byte  `json:"proxy_protocol,omitempty"`
```

Comment:
Is `*byte` really necessary? If the API server doesn't include one, it will be `false` anyways (Go's zero-initialization rule).

Reply:
Go never complains on missing inputs keys, plus this is `AuthResponse` which sshmux isn't supposed to *emit*, only receive.